### PR TITLE
Add de.linuxguides.RechnungsAssistent

### PR DESCRIPTION
**Pull request by Jean28518**
**Notice: This app is only available in german language.** I didn't find any special instructions for that circumstance.

Git Repo: https://github.com/Jean28518/invoice-creator-german

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am the upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: de.linuxguides.RechnungsAssistent.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - install -D rechnungs-assistent.sh /app/bin/rechnungs-assistent.sh
+      - cp -r data /app/bin/data
+      - cp -r html /app/bin/html
+      - cp -r lib /app/bin/lib
+      - cp -r chromium /app/bin/chromium
+    sources:
+      - type: archive
+        url: https://github.com/Jean28518/invoice-creator-german/releases/download/v0.2.0/rechnungs-assistent-bundle-0.2.0.zip
+        sha256: 1e99c33b5c1700d95d907a9b7bbecb0c7f6bb9c32aafe2808c42b3b66c54c172
+finish-args:
```
Comment (hfiguiere) [inline de.linuxguides.RechnungsAssistent.yml:22]: finish args is usually above modules.

File: de.linuxguides.RechnungsAssistent.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+app-id: de.linuxguides.RechnungsAssistent
```
Comment (hfiguiere) [inline de.linuxguides.RechnungsAssistent.yml:1]: `linuxguides.de` doesn't see to exist.

File: de.linuxguides.RechnungsAssistent.desktop
[the remark this answers is not in the document]
Comment (hfiguiere) [reply de.linuxguides.RechnungsAssistent.desktop:1]: also upstream has a .desktop file so you shouldn't have one here.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
there is also no icon installed.

File: de.linuxguides.RechnungsAssistent.yml (hunk 1 above)
Comment (Jean28518) [reply de.linuxguides.RechnungsAssistent.yml]: Thanks!

File: de.linuxguides.RechnungsAssistent.desktop
[the remark this answers is not in the document]
Comment (Jean28518) [reply de.linuxguides.RechnungsAssistent.desktop:1]: Check, thank you

**Comment by Jean28518** [comment]
Thank you very much for your hints.
I hope I fixed everything.
Should I create a new PR from new-pr or is this okay?

File: de.linuxguides.RechnungsAssistent.yml (hunk 2 above)
Comment (hfiguiere) [reply de.linuxguides.RechnungsAssistent.yml:1]: yeah that's ok. it didn't work when I checked.

**Comment by hfiguiere** [comment]
bot, build de.linuxguides.RechnungsAssistent

File: de.linuxguides.RechnungsAssistent.metainfo.xml
Comment (hfiguiere) [inline de.linuxguides.RechnungsAssistent.metainfo.xml:1]: this file is never installed.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
no desktop file is installed either.

**Comment by Jean28518** [comment]
Thank you very much for your patience.
Please retry.

**Comment by Jean28518** [comment]
bot, build de.linuxguides.RechnungsAssistent

Thanks. Appdata in metainfo.xml is now in english.

**Comment by Jean28518** [comment]
bot, build de.linuxguides.RechnungsAssistent

**Comment by Jean28518** [comment]
bot, build de.linuxguides.RechnungsAssistent

**Comment by Jean28518** [comment]
From my side everything is done now.

**Pull request closed without merging**